packages/runtime/container-runtime-definitions/src/snapshot.ts

```diff
@@ -0,0 +1,56 @@
+/*!
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { ISnapshotTree } from "@fluidframework/protocol-definitions";
+
+export type PropertyValues<T> = T[keyof T];
+
+export const containerSnapshotFormatVersions = {
+    missing: undefined,
```

Comment:
can you please add some comments on what these fields mean?
Also should be defined interface properly (with all of the comments that go to documentation) then derive it from this value?

Reply:
Added comments.

Not sure what exactly you mean by interface.. If I add an interface, I need to list the versions/types twice I think?
I have changed to more simple union type.

The alternative I could think (without listing all versions twice) was the same as it was before but change `current` to `v1` and `next` to `v2` etc. so that we don't have to update all references every time.